game/view/pickwidget.lua

```diff
@@ -1,7 +1,7 @@
 
 local RES = require 'resources'
 local DEFS = require 'domain.definitions'
-local COLORS = require 'domain.definitions.colors'
```

Comment:
Remove this from the PR because I fixed it in another PR just for minor fixes like this.

Reply:
Done